src/brsTypes/BrsType.ts

```diff
@@ -123,13 +125,13 @@ export function getBrsValueFromFieldType(type: string, value?: string): BrsType
         case "float":
             returnValue = value ? new Float(Number.parseFloat(value)) : new Float(0);
             break;
-        case "roArray":
+        case "roarray":
         case "array":
-            returnValue = BrsInvalid.Instance;
+            returnValue = new RoArray([]);
```

Comment:
This still doesn't technically do everything it should (it ignores the `value` parameter and always creates empty arrays/associative arrays). But it unblocks the scenario I put in the description for this PR, and it gets us _closer_ to complete.

Reply:
So is the comment on line 105 still accurate?  "...Objects such as array/AA aren't handled at the moment."

Reply:
Good callout! I can edit that comment to be more clear that they're partially handled, but not fully 👍